be/src/olap/base_compaction.cpp

```diff
@@ -28,6 +28,13 @@ BaseCompaction::BaseCompaction(TabletSharedPtr tablet, const std::string& label,
 BaseCompaction::~BaseCompaction() { }
 
 OLAPStatus BaseCompaction::compact() {
+    RETURN_NOT_OK(prepare_compact());
```

Comment:
Who will call `BaseCompaction::compact()` now?

Reply:
> Who will call `BaseCompaction::compact()` now?

`BaseCompaction::compact()` will still be called in "be/src/http/action/compaction_action.cpp" for `/api/compaction/run`.